anyric/super-record
# #167806283 Create Data Migration for Setup Defaults

#### What does this PR do?
This PR create seed data for use as defaults in the database to enable login and setup of the system

#### Description of Task to be completed?
- Create seed data for use as defaults in the database

#### How should this be manually tested?
- create new empty database and update the database name value in the .env file
- Clone this branch
- In the root directory run `python manage.py makemigrations`
- In the root directory run `python manage.py migrate`
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to login
- Login with default admin details`username=admin` and `password=1234@admin`
- Browse the links to verify you can access and configure the system

#### Any background context you want to provide?
Currently when the project is cloned by a new developer and runs, they can't login since there is no provision for registering new users without login by an admin or manager role. There should be a default user with admin role to facilitate the initial application setups.

#### What are the relevant pivotal tracker stories?
[#167806283](https://www.pivotaltracker.com/story/show/167806283)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
